.copyrite_aliases

```diff
@@ -113,5 +113,10 @@
     "mails": ["tushar.sadhwani000@gmail.com", "tushar@deepsource.io"],
     "authoritative_mail": "tushar.sadhwani000@gmail.com",
     "name": "Tushar Sadhwani"
+  },
+  {
+    "mails": ["jacobtylerwalls@gmail.com"],
```

Comment:
This is used to recognize two mails as the same person. Unless you use other emails sometime this is not necessary.